tests/test_pkg_config.py

```diff
@@ -33,10 +33,8 @@ def test_load_default(self):
         self.assertIn("Default", settings["QueryDefinitions"])
 
         for path in settings["QueryDefinitions"]["Default"]:
-            self.assertTrue(Path(path).is_dir())
```

Comment:
Is the issue here that the path resolves to something unreachable?
can we fix it by prefixing the returned path with 'msticpy'? Maybe we should move the tests for data-specific checks into the test_dataqueries.

Reply:
Fixed this in a number of new assertions. I have also added an element to test_dataqueries but feel this config setup element probably still sits best here, at least for now.